### PR TITLE
JAX-WS TCK minor documentation fixes

### PR DESCRIPTION
**Pull request by rfelcman**
This PR contains some minor fixes in JAX-WS documentation.
Bug #435 was verified there. Only Ant location had to be changed.

Signed-off-by: Radek Felcman <radek.felcman@oracle.com>

### CODE REVIEW

File: install/jaxws/docs/JAXWS3.0-ReleaseNotes.html [hunk 1]
```diff
@@ -183,7 +183,7 @@ <h2><a name="install_config_tomcat8">Quick Steps to Configure Apache Tomcat 8 fo
               <li>cp /jaxws-ri/lib/*.jar /apache-tomcat-10.0.x/shared/lib</li>
               <li>cd /apache-tomcat-10.0.x/shared/lib; &nbsp; mv jaxws-api.jar jaxb-api.jar /apache-tomcat-10.0.x/endorsed</li>
               <li>cp /jaxwstck/lib/jaxwstck.jar /jaxwstck/lib/tsharness.jar /apache-tomcat-10.0.x/shared/lib</li>
-              <li>cd /jaxwstck/bin; &nbsp; mv ts.jte ts.jte.javaee8; &nbsp; cp ts.jte.tomcat.jdk18 ts.jte</li>
+              <li>cd /jaxwstck/bin; &nbsp; mv ts.jte ts.jte.javaee9; &nbsp; cp ts.jte.tomcat.jdk18 ts.jte</li>
```
Comment (scottmarlow) [inline install/jaxws/docs/JAXWS3.0-ReleaseNotes.html:186]: Should `javaee9` be changed to `jakartaee9`?

File: user_guides/jaxws/src/main/jbake/content/config.adoc [hunk 2]
```diff
@@ -416,8 +416,8 @@ path entries (`;` on Windows, `:` on UNIX/Linux).
 On Windows, you must escape any backslashes with an extra backslash in
 path separators used in any of the following properties, or use forward
 slashes as a path separator instead. For example, if the Jakarta EE 9 Compatible Platform
-installation is `C:\JavaEE8`, you must specify it as `C:\\JavaEE8` or
-`C:/JavaEE8`.
+installation is `C:\JavaEE9`, you must specify it as `C:\\JavaEE9` or
+`C:/JavaEE9`.
```
Comment (scottmarlow) [inline user_guides/jaxws/src/main/jbake/content/config.adoc:421]: Same here, should `JavaEE9` be changed to `JakartaEE9`? 

Are there external dependencies on the `javaee9' + `JakartaEE9` names?

File: install/jaxws/docs/JAXWS3.0-ReleaseNotes.html [hunk 3]
```diff
@@ -167,7 +167,7 @@ <h2><a name="install_setup_run">Installing, Setting Up, and Running the Jakarta
         The online version of the JT Harness version 5.0 documentation is available <a href="https://wiki.openjdk.java.net/display/CodeTools/Documentation">here</a>.</p>
 
         <hr>
-        <h2><a name="install_config_tomcat8">Quick Steps to Configure Apache Tomcat 8 for Jakarta XML Web Services TCK</a></h2>
+        <h2><a name="install_config_tomcat10">Quick Steps to Configure Apache Tomcat 10 for Jakarta XML Web Services TCK</a></h2>
```
Comment (scottmarlow) [inline install/jaxws/docs/JAXWS3.0-ReleaseNotes.html:170]: :+1:

File: install/jaxws/docs/JAXWS3.0-ReleaseNotes.html [hunk 4]
```diff
@@ -50,7 +50,7 @@ <h2><a name="kit_contents">Kit Contents</a></h2>
...
    <ul>
       <li>A <strong>Signature test</strong> that checks that all of the public APIs are supported/defined as specified in the Jakarta XML Web Services Version 3.0 implementation that is being tested.</li>
-      <li><strong>API tests</strong> to verify the functionality of all the public APIs under <code>jakarta.xml.ws</code>, <code>jakarta.xml.ws.handler</code>, <code>jakarta.xml.ws.handler.soap</code>, <code>jakarta.xml.ws.http</code>, <code>jakarta.xml.ws.spi</code>, <code>jakarta.xml.ws.spi.http</code>, <code>jakarta.xml.ws.soap</code>,  <code>jakarta.xml.ws.wsaddressing</code>, <code>jakarta.jws</code>, and <code>jakarta.jws.soap</code>.</li>
+      <li><strong>API tests</strong> to verify the functionality of all the public APIs under <code>jakarta.xml.ws</code>, <code>jakarta.xml.ws.handler</code>, <code>jakarta.xml.ws.handler.soap</code>, <code>jakarta.xml.ws.http</code>, <code>jakarta.xml.ws.spi</code>, <code>jakarta.xml.ws.spi.http</code>, <code>jakarta.xml.ws.soap</code>,  <code>jakarta.xml.ws.wsaddressing</code>, <code>jakarta.jws</code> and <code>jakarta.jws.soap</code>.</li>
```
Comment (lukasj) [inline install/jaxws/docs/JAXWS3.0-ReleaseNotes.html:53]: this change reminds me there is a list of packages somewhere in the guide (section 3 or 4 I think) which is missing `jakarta.jws.*` packages. Can you add them there so the list is consistent, please?

Comment (rfelcman) [reply install/jaxws/docs/JAXWS3.0-ReleaseNotes.html:53]: It's about 1.2.2 and 2.5.

**Review by lukasj — APPROVED** [review]
yup, this looks fine to me

**Pull request merged**